test/index.js

```diff
@@ -1652,15 +1659,19 @@ suite ('laws', () => {
   });
 
   suite ('RegExp', () => {
-    const arb = jsc.string.smap (s => {
-      const validFlags = ['', 'g', 'i', 'm', 'gi', 'gm', 'im', 'gim'];
-      const flags = validFlags[Math.floor (Math.random () * validFlags.length)];
+    const flagsArb = jsc.oneof (['', 'g', 'i', 'm', 'gi', 'gm', 'im', 'gim'].map (jsc.constant));
+    const patternArb = jsc.suchthat (jsc.string, s => {
       try {
-        return new RegExp (s, flags);
-      } catch (err) {
-        return /(?:)/;
```

Comment:
Is `/(?:)/` no longer necessary because `jsc.suchthat` rejects strings that are not regular expressions?

Reply:
Yes.